en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+    
+    ``` tab="If the preferred channel is EMAIL" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "EMAIL"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    
+    ``` tab="If the Preferred channel is SMS" 
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username":"kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/preferredChannel","value": "SMS"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "94774747551"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+    ```
+    
+    !!! note
+        for more information, see 
+        [API Requests and Responses](.././learn/extended-self-registration-api-and-account-confirmation-api/#api-requests-and-responses).
+   
+2.  Following response will be returned by the API.
+
+    ``` tab="Preferred channel as EMAIL" 
+    {
+        "code":"USR-02001",
+        "message":"Successful user self registration. Pending account verification.",
+        "notificationChannel":"EMAIL",
+        "confirmationCode":null
+    } 
+    ```
+    
+    ``` tab="Preferred channel as SMS" 
+    {
+        "code":"USR-02001",
+        "message":"Successful user self registration. Pending account verification.",
+        "notificationChannel":"SMS",
+        "confirmationCode":null
+    } 
+    ```
+    
+    !!! Note
+        - If the preferred channel is not specified in the request, the server will decide the 
+        notification channel if the `enable_resolve_notification_channel` property is `enabled`.
+        Notified channel will be returned in the `notificationChannel`.
+        
+        - **NOTE:** If the above property is not enabled the user will be notified via 
+        the channel specfied by `default_notification_channel` 
+        proerty in `deployement.toml`.
+        
+        - For more information, see [Notification channel selection criteria](../.
+        ./learn/extended-self-registration-api-and-account-confirmation-api/#notification-channel-selection-criteria).
+        
+3. If the notification channel is **EMAIL**, navigate to the email address and and click 
+the button or the confirmation link. Then the user account will be unlocked.
```

Comment:
If the notification channel used is **EMAIL**, **access the relevant email account** and click the button or the confirmation link.

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549